### DISCUSSION TITLE
While connected to OpenVPN Adguard not responding to DNS queries which = no connectivity

### DISCUSSION THREAD

**Discussion by privacyguy123** [AdguardTeam/AdGuardHome#6168] [Q&A]
In my OpenVPN conf I have ```pull-filter ignore dhcp-option DNS``` & ```dhcp-option DNS 192.168.0.100``` specified yet no connectivity. It seems my Adguard instance is not replying to private IP like ```10.x.x.x``` which is what I'm  being assigned by OpenVPN.

How do I fix this?

**Comment by AaronIsMyBrother** [comment]
I would like to ask you a question. Did you try to start AG with not 'root' user?
